core/src/main/java/org/apache/accumulo/core/iterators/SortedKeyValueIterator.java

```diff
@@ -147,4 +147,21 @@
    *              if not supported.
    */
   SortedKeyValueIterator<K,V> deepCopy(IteratorEnvironment env);
+
+  /**
+   * Closes any resources that were opened by the <tt>SortedKeyValueIterator</tt>. This method does nothing by default. The implementing class must close its
+   * <tt>SortedKeyValueIterator source</tt>. This will be provided in the <tt>init</tt> method or by the constructor.
+   *
+   * @exception UncheckedIOException
```

Comment:
After revisiting these changes, I am now being reminded as to why I abandoned them months ago...  @keith-turner may have an alternate solution that is far less widespread and more manageable. See https://issues.apache.org/jira/browse/ACCUMULO-1280